Robot2019/src/main/java/frc/robot/subsystems/Lights.java

```diff
@@ -16,44 +17,58 @@
  * Add your docs here.
  */
 public class Lights extends Subsystem {
+
+  public enum LightState {
+    OFF,
+    CARGO,
+    HATCH,
+  }
   // Put methods for controlling this subsystem
   // here. Call these from Commands.
-  private VictorSP lights;
-  private double lightsValue = 0;
+  private Relay lights;
+  private LightState lightsState;
 
   /**
    * 0 is off 0.5 is orange 1 is yellow
    */
 
-  public Lights(VictorSP lights) {
+  public Lights(Relay lights) {
     this.lights = lights;
```

Comment:
I *think* you also need to call lights.setDirection(Relay.Direction.kBoth) so that you can use kOff, kForward, and kReverse.